install/ossm-installing-openshift-service-mesh.adoc

```diff
@@ -39,7 +39,9 @@ include::modules/ossm-about-bookinfo-application.adoc[leveloffset=+1]
 include::modules/ossm-deploying-bookinfo-application.adoc[leveloffset=+2]
 include::modules/ossm-about-accessing-bookinfo-application-using-gateway.adoc[leveloffset=+2]
 include::modules/ossm-accessing-bookinfo-application-using-istio-gateway-injection.adoc[leveloffset=+2]
-include::modules/ossm-accessing-bookinfo-application-using-gateway-api.adoc[leveloffset=+2]
+//include::modules/ossm-accessing-bookinfo-application-using-gateway-api.adoc[leveloffset=+2] 
```

Comment:
It isn't necessary to omit the entire module; most of it is still applicable on OCP 4.19.  Only the first note (about CRDs) and the first step in the procedure ("Enable the Gateway API CRDs") are inapplicable for OCP 4.19 onward.  

This is what the documentation should say:

- For OCP 4.18 and earlier, you (the cluster admin) are responsible for installing the CRDs (using the currently documented step).  
- For OCP 4.19 and later, OCP itself is responsible for installing and updating the CRDs, and you (the cluster admin) are not permitted to create, update, or delete these CRDs.